testdriver/press-keys.yaml

```diff
@@ -13,5 +13,12 @@ steps:
         keys:
           - command
           - t
-      - command: assert
-        expect: a new tab was created
+      - command: wait-for-text
+        text: "Learn more"
+      - command: press-keys
+        keys:
```

Comment:
Since we're on `linux`, +Shift+→ doesn't seem to do anything.

So instead I did a 3-key combo to open devtools.